ios/DomainLayer/SharedDomain/Sources/SharedDomainMocks/Extensions/UseCaseStreamStreamMock.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Created by David Kadlček on 27.02.2023
+//  Copyright © 2023 Matee. All rights reserved.
+//
+
+import Foundation
+import DevstackKmpShared
+
+open class UseCaseFlowNoParamsMock<Out>: UseCaseFlowNoParams {
```

Comment:
Wrong file name `UseCaseStreamStreamMock` .. should be `UseCaseFlowNoParamsMock`
Also, I'm not sure about the folder naming, these are not extensions, so maybe name it something like `KMPUtilities`? And you can put that `TestError` there as well 🙂

Reply:
Done